### ISSUE TITLE
Playing audio in mobile safari opens a full screen player

### ISSUE THREAD

**Issue by babenzele** [CookPete/react-player#188]
Playing audio in mobile safari opens a full screen player. I've tried `playsInline` with no luck.

**Comment by babenzele** [comment]
Since my url is a streaming link, `AUDIO_EXTENSIONS.test(url)` returns `false` and therefore defaults to `video`.

http://s2.voscast.com:9208/;&type=mp3

I took the `.` out of the regex to make this work, but if I made a `StreamingAudio` component, would that be something that the maintainers of this library would want to be added?

**Comment by CookPete** [comment]
>if I made a StreamingAudio component, would that be something that the maintainers of this library would want to be added?

It doesn't make sense to add another new component just to fix this. I think the best option is to add a `forceAudio` property to the `fileConfig` prop object, to be used like so:

```js
<ReactPlayer
  url='http://s2.voscast.com:9208/;&type=mp3'
  playsinline
  fileConfig={{ forceAudio: true }}
/>
```

**Comment by babenzele** [comment]
Good thinking, I'll take a swing at it.

**Comment by CookPete** [comment]
No need! Published in `0.17.2`

**Comment by babenzele** [comment]
Right on!
